app/models/package_manager/go.rb

```diff
@@ -280,6 +300,11 @@ def self.encode_for_proxy(str)
       str.gsub(/[A-Z]/) { |s| "!#{s.downcase}" }
     end
 
+    # Returns the types listed at the top of pkg.go.dev pages. Known values are: module, package, directory, command.
+    private_class_method def self.page_types(raw_project: nil)
```

Comment:
Why default to nil here? I think we need nil protection if we are not requiring `raw_project` to actually be passed in.